saleor/graphql/discount/types.py

```diff
@@ -64,6 +66,9 @@ class Voucher(CountableDjangoObjectType):
     countries = graphene.List(
         CountryDisplay,
         description='List of countries available for the shipping voucher.')
+    translation = graphene.Field(
+        VoucherTranslation, language_code=graphene.String(required=True),
+        description='Translation.', resolver=resolve_translation)
```

Comment:
We could improve the description here as well, maybe something like: "Returns translated Sale fields for the given language code."